dart_conf_flutter/lib/src/schedule.dart

```diff
@@ -131,9 +131,8 @@ class _SchedulePageState extends State<SchedulePage> {
       snapshot['title'],
       snapshot['description'],
       dateTime,
-      duration == null ? null : new Duration(minutes: snapshot['duration']),
-      presenters:
-          (snapshot['authors'] as String).split(',').map((s) => s.trim()),
+      duration == null ? 30 : new Duration(minutes: snapshot['duration']),
```

Comment:
For this line, what about `new Duration(minutes: snapshot['duration'] ?? 30)`?

Reply:
done, and added a static const for the default duration